applications/DAOsign.md

```diff
@@ -0,0 +1,183 @@
+
+**Oleksandra Burmenska:** Director of Business Development and Partnerships
+Background:
+Engineering, MBA, and Psychology diplomas. Over a decade of experience in IT: development and managerial positions, last five years focused on the Blockchain industry. 
+Experience with enterprises and startup companies, scaling teams, and building blockchain products.
+Previous blockchain projects include:
+- successfully managed ICOs during 2017-2019 (example: Õpet – Medium ) 
+- contribution to Polywrap, NEAR, Axelar, GnosisSafe, Avalanche ecosystems
+- contribution to nodle.com ecosystem: managing the block explorer development (before it moved to Subscan), leading the Landslide - Avalanche subnet development,etc. 
+
+**Misha Kushka**: Tech Lead and Blockchain Developer
+Background: 6+ years of professional experience as a developer, 4+ years of professional experience in the Blockchain field, 4+ years experience as a lead
+
+### Team Code Repos
+
+- https://github.com/ConsiderItDone
+- https://github.com/DAOsign
+
+Please also provide the GitHub accounts of all team members. If they contain no activity, references to projects hosted elsewhere or live are also fine.
+
+- [Eugene Fine](https://github.com/ConsiderItDone)
+- [Ramil Amerzyanov](https://github.com/ramilexe)
+- [Oleksandra Burmenska](https://github.com/Burmenska)
+- [Misha Kushka](https://github.com/kushkamisha)
+
+### Team LinkedIn Profiles (if available)
+
+- [Eugene Fine](https://www.linkedin.com/in/eugenefine/)
+- [Ramil Amerzyanov](https://www.linkedin.com/in/ramil-amerzyanov/)
+- [Oleksandra Burmenska](https://www.linkedin.com/in/oleksandra-burmenska/)
+- [Misha Kushka](https://www.linkedin.com/in/mkushka/) 
+
+
+## Development Status :open_book:
+
+We are currently working on a tech demo (kind of a pre-beta version) and non-blockchain version of DAOsign is available here: https://testnet.daosign.org
+
+## Development Roadmap :nut_and_bolt:
+
+**Overview**
+
+Total Estimated Duration: 2 months
+Full-Time Equivalent (FTE): 2 FTE per month
+Total Costs: 48,100 USD
+
+Milestone 1 - Smartcontracts. 
+Total Costs: 40,300 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License |  Apache 2.0 |
+| 0b. | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can upload/generate the agreement, add signers/observers and complete the signature process with storing signature proofs either in test or mainnet, which will show how the new functionality works. |
+| 0c. | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| 0d. | Delivery format | We will provide a website that will include the agreement creation, modification and signing functionality for users with further proofs storing on Polkadot chain.
```

Comment:
`0d` is usually a Dockerfile or some other setup automation tool to help us and others test the project. Ideally, this would run a local node, deploy and instantiate the contracts or run unit tests on the smart contracts.

Reply:
@semuelle @keeganquigley added docker step as `0d`

Reply:
@keeganquigley I don't see how docker could help with npm/yarn modules. It will be used docker from previous milestone to spin up node with smart contract

Reply:
@ramilexe I just realised that M2 still doesn't contain the Docker deliverable. Could you please add it? As already mentioned by @keeganquigley it's sufficient if it's the same as in M1, but I think it'd make it easier to evaluate it you'd deliver.

Reply:
@takahser @keeganquigley I thought we agreed we don't need docker for M2 because it will contain Javascript library and frontend that usually delivered as a npm module, not docker container

Reply:
@ramilexe I think the idea was to use the same docker config file(s) as in M1 to test the integration with the js library. You can add a note that indicates that. Alternatively, you can add a note to the SDK indicating that it will be compatible with the docker deliverable from M1.

Reply:
Added docker to m2. Is it ok?